src/components/document/sort-work-document-area.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import classNames from "classnames";
+import { useAppConfig, useProblemStore,
+  usePersistentUIStore, useUserStore, useClassStore, useUIStore, useStores } from "../../hooks/use-stores";
+import { DocumentModelType } from "../../models/document/document";
+import { EditableDocumentContent } from "./editable-document-content";
+import { getDocumentDisplayTitle } from "../../models/document/document-utils";
+import EditIcon from "../../clue/assets/icons/edit-right-icon.svg";
+import { observer } from "mobx-react";
+import { tabName } from "./sort-work-view";
```

Comment:
This is a circular reference. sort-work-view is importing sort-work-document-area to be able to render the SortWorkDocumentArea component. And then here sort-work-document-area is importing sort-work-view to get the tabName. 

If you update ENavTab to include sort-work, then you can use that instead of tabName from sort-work-view.